app/src/main/java/com/shopapp/ui/account/AccountFragment.kt

```diff
@@ -157,15 +135,32 @@ class AccountFragment :
     override fun showContent(data: Boolean) {
         super.showContent(data)
         if (data) {
-            settingsMenuItem?.isVisible = true
             presenter.getCustomer()
         } else {
-            settingsMenuItem?.isVisible = false
-            unauthGroup.visibility = View.VISIBLE
-            authGroup.visibility = View.GONE
+            changeViewState(false)
         }
     }
 
+    private fun changeViewState(isCustomerAuthorized: Boolean, customer: Customer? = null) {
+        val oldFragment: Fragment? = childFragmentManager.findFragmentById(R.id.accountContainer)
+        val newFragment: Fragment = if (isCustomerAuthorized && customer != null) {
+            AccountAuthFragment.newInstance(customer)
+        } else {
+            AccountUnAuthFragment.newInstance(shop)
+        }
+
+        if (newFragment.javaClass != oldFragment?.javaClass) {
+            childFragmentManager.beginTransaction()
```

Comment:
Can we sync API for replacing fragment with `FragmentManager.replaceOnce`? We have 2 different approaches - in the extension we don't need to create a transaction (it created by our method automatically), but in this case, we need to do this. 

I think we can create similar method `replace` which also will create transaction.